ui/pages/confirmations/components/simulation-details/fiat-display.test.tsx

```diff
@@ -1,47 +1,104 @@
 import React from 'react';
 import { screen } from '@testing-library/react';
 import configureStore from 'redux-mock-store';
+import { merge } from 'lodash';
 import { useFiatFormatter } from '../../../../hooks/useFiatFormatter';
 import { renderWithProvider } from '../../../../../test/lib/render-helpers';
 import mockState from '../../../../../test/data/mock-state.json';
+import { CHAIN_IDS } from '../../../../../shared/constants/network';
 import { IndividualFiatDisplay, TotalFiatDisplay } from './fiat-display';
 import { FIAT_UNAVAILABLE } from './types';
 
-const store = configureStore()(mockState);
+const mockStateWithTestnet = merge({}, mockState, {
+  metamask: {
+    providerConfig: {
+      chainId: CHAIN_IDS.SEPOLIA,
```

Comment:
If we are going with this approach, I believe it's crucial to set some state props like this for the branch of these tests. 

Even if `mockState` is set as using Goerli chain id which is fine for this test, I believe we should still set the stage for only our test. This is the reason of I am creating `mockStateWithTestnet`